x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/steps/components/package_policy_input_panel.tsx

```diff
@@ -139,9 +140,9 @@ export const PackagePolicyInputPanel: React.FunctionComponent<{
               label={
                 <EuiFlexGroup alignItems="center" gutterSize="s">
                   <EuiFlexItem grow={false}>
-                    <EuiText>
-                      <h4 id={titleElementId}>{packageInput.title || packageInput.type}</h4>
-                    </EuiText>
+                    <EuiTitle size="xs">
+                      <h3 id={titleElementId}>{packageInput.title || packageInput.type}</h3>
+                    </EuiTitle>
```

Comment:
I opted to switch this out for `<EuiTitle/>` because I could pass a size prop and ensure the headings were the same size as other H3s on the page without custom CSS.